LibreNMS/OS/Siteboss530.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Siteboss530.php
+ *
+ * Asentria Siteboss 530
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @link       http://librenms.org
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Interfaces\Discovery\OSDiscovery;
+use LibreNMS\OS;
+
+class Siteboss530 extends OS implements OSDiscovery
+{
+    public function discoverOS(): void
+    {
+        $device = $this->getDeviceModel();
```

Comment:
Sorry this needs to be updated, check another OS for examples, was recently changed

Reply:
whoops, I updated the siteboss550 but forgot this one. Why did this not show an error in Travis?

Reply:
these two similar devices have the exact same discoverOS code. Is there a way to use `group_os` with the new OS polling/discovery method? Similar to `includes/discovery/<group_os>.inc.php`

Reply:
Don't know if group exists, but you could just make it extend the other one, of create one base one if it doesn't already exists. 

class Siteboss530 extends Siteboss550 or so

Reply:
I refactored code into advanced discovery, `includes/discovery/sensors/*/siteboss.inc.php` and merged the two OS back into one